website/docs/d/cloud_backup_schedule.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: cloud_backup_schedule"
+sidebar_current: "docs-mongodbatlas-datasource-cloud-backup-schedule"
+description: |-
+    Provides a Cloud Backup Schedule Datasource.
+---
+
+# mongodbatlas_cloud_backup_schedule
+
+`mongodbatlas_cloud_backup_schedule` provides a Cloud Backup Schedule datasource. An Atlas Cloud Backup Schedule provides the current cloud backup schedule for the cluster. 
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
+
+## Example Usage
+
+```hcl
+resource "mongodbatlas_cluster" "my_cluster" {
+  project_id   = "<PROJECT-ID>"
+  name         = "clusterTest"
+  disk_size_gb = 5
+
+  //Provider Settings "block"
+  provider_name               = "AWS"
+  provider_region_name        = "EU_CENTRAL_1"
+  provider_instance_size_name = "M10"
+  cloud_backup     = true // enable cloud backup snapshots
+  provider_disk_iops          = 100
+}
+
+resource "mongodbatlas_cloud_backup_schedule" "test" {
```

Comment:
can we add an example with policies here

Reply:
already there in docs of resource